Lib/test/test_unittest/test_result.py

```diff
@@ -275,6 +275,39 @@ def get_exc_info():
         self.assertEqual(len(dropped), 1)
         self.assertIn("raise self.failureException(msg)", dropped[0])
 
+    def test_addFailure_filter_traceback_frames_chained_exception_cycle(self):
+        class Foo(unittest.TestCase):
+            def test_1(self):
+                pass
+
+        def get_exc_info():
+            try:
+                # Create two directionally opposed cycles
+                # __cause__ in one direction, __context__ in the other
+                # Only one is needed since they serve the same role in _clean_tracebacks()
+                # But this way we cover both possibilities if the code changes
```

Comment:
```suggestion
```

(not necessary)

Reply:
@iritkatriel thank you for your feedback. Just to clarify, are you saying that the comment itself isn't necessary or are you saying that the approach isn't necessary?

Reply:
The comment.

Reply:
Addressed in 8388d82